### PR TITLE
Breaking change API mod-calendar MODCAL-57

### PR DESCRIPTION
**Pull request by OleksiiMaksymov**
## Purpose
A new status code has been added to the module for the `/calendar/periods/{servicePointId} /period/{periodId}` endpoint.

Resolve: [MODCAL-57](https://issues.folio.org/browse/MODCAL-57)

## Approach
Raising the mod-calendar version in the descriptor.

## Pre-Merge Checklist:
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.

### CODE REVIEW

File: descriptors/ModuleDescriptor-template.json [hunk 1]
```diff
@@ -715,7 +715,7 @@
     },
     {
       "id": "calendar",
-      "version": "3.0"
+      "version": "4.0"
```
Comment (marcjohnson-kint) [inline descriptors/ModuleDescriptor-template.json]: Given that there are no functional code changes associated with this version compatibility change, does that mean that circulation could work with either `calendar 3.0` or `calendar 4.0`?

Comment (OleksiiMaksymov) [reply descriptors/ModuleDescriptor-template.json]: The mod-circulation could work with either calendar 3.0 or calendar. Сhanges in the mod-calendar are made only for the UI.

Comment (marcjohnson-kint) [reply descriptors/ModuleDescriptor-template.json]: Then please use dual support in this change

Comment (OleksiiMaksymov) [reply descriptors/ModuleDescriptor-template.json]: Fix

**Comment by marcjohnson-kint** [comment]
This change should be associated with an issue in the mod-circulation JIRA project, for release management.

**Pull request merged**